core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerFacade.java

```diff
@@ -139,8 +138,14 @@ private void setMovieAudioAttributes(SimpleExoPlayer exoPlayer) {
         }
     }
 
-    private void attachToSurface(SurfaceHolder surfaceHolder) {
-        exoPlayer.setVideoSurfaceHolder(surfaceHolder);
+    private void attachToSurface(PlayerSurfaceHolder playerSurfaceHolder) {
+        if (playerSurfaceHolder.containsSurfaceView()) {
```

Comment:
I would prefer to avoid this check, maybe better to have something like `playerSurfaceHolder.install(exoPlayer)`

Reply:
ooooh yeah, nice! I'll do that